### DISCUSSION TITLE
How to set a custom Tinker file location?

### DISCUSSION THREAD

**Discussion by walkonthemarz** [Roboroads/laravel-tinker#355] [Q&A]
The default Tinker file is located at JetBrains ${Configuration directory﻿}/laravel-tinker dir.
Is there a way to modify the location?

**Comment by Roboroads** [comment]
Something like this is requested in the issues tab. I am thinking of a way to enable this=

**Comment by Roboroads** [comment]
This is now possible in the v3 update - which is pending!
